### DISCUSSION TITLE
Manual Installation, VNC viewer not working after Visualizer install

### DISCUSSION THREAD

**Discussion by vHyrax** [onlaj/Piano-LED-Visualizer#341] [Q&A]
I'm pretty new to this, just wondering if anyone could help me. When I was working on step 7 on the manual installation which was the actual Piano Led Visualizer, I was able to install it good. The SPI interface became functional and everything works for this. But right after installing this, I only get a display of "Cannot currently show the desktop" on VNC Viewer. Can someone help me with a solution??

**Comment by onlaj** [comment]
Hello.
I'm not sure what could be causing this, but maybe just use SSH to connect to the Raspberry Pi?

**Reply by vHyrax** [reply]
Yep, I use PuTTY for SSH connection. That's still working good. VNC was working just fine until I installed the Visualizer unfortunately. My project is trying to get a DAW/VST on the raspberry pi that'll add a great supplement with the visualizer ;)
